cypress.config.js

```diff
@@ -10,10 +10,17 @@ module.exports = defineConfig({
           const email = faker.internet.email();
           const randomNumber = Math.floor(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + randomNumber,
+            username: faker.person.firstName() + randomNumber,
             email: email.toLowerCase(),
             password: '12345Qwert!'
```

Comment:
Hardcoding passwords is a security risk and is generally not a good practice. It would be better to generate a password dynamically, similar to how you generate the username and email.